test/gaudi_opts/geoTest_cld.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2019-2023 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from Gaudi.Configuration import *
+
+from Configurables import TrackingCellIDEncodingSvc, GeoSvc
+algList = []
+svcList = []
+
+geoservice = GeoSvc("GeoSvc")
+geoservice.detectors = [os.environ["K4GEO"]+"/CLIC/compact/CLIC_o3_v15/CLIC_o3_v15.xml"]
+geoservice.OutputLevel = INFO
+geoservice.EnableGeant4Geo = False
+svcList.append(geoservice)
+
+cellIDSvc = TrackingCellIDEncodingSvc("CellIDSvc")
+cellIDSvc.EncodingStringParameterName = "GlobalTrackerReadoutID"
+cellIDSvc.OutputLevel = DEBUG
+svcList.append(cellIDSvc)
+
+from Configurables import ApplicationMgr
+ApplicationMgr( TopAlg = algList,
+                EvtSel = 'NONE',
+                EvtMax = 3,
+                ExtSvc = svcList,
```

Comment:
@hegner: What is this `ExtSvc` actually for? Does the list of services have to go here, or just specific ones?

Reply:
This lists all the extra services (hence the name) that should be made available to the application manager

Reply:
I wasn't sure if Ext means "External". But that means the services created have to be added to this list, right?

Reply:
Yes. Otherwise the svcLocator will not know them properly